go/serialization.go

```diff
@@ -352,6 +350,77 @@ func SerializeSCTSignatureInput(sct SignedCertificateTimestamp, entry LogEntry)
 	}
 }
 
+func serializeV1SCT(sct SignedCertificateTimestamp) ([]byte, error) {
+	if err := checkExtensionsFormat(sct.Extensions); err != nil {
+		return nil, err
+	}
+	var buf bytes.Buffer
+	if err := binary.Write(&buf, binary.BigEndian, V1); err != nil {
+		return nil, err
+	}
+	if err := binary.Write(&buf, binary.BigEndian, sct.LogID); err != nil {
+		return nil, err
+	}
+	if err := binary.Write(&buf, binary.BigEndian, sct.Timestamp); err != nil {
+		return nil, err
+	}
+	if err := writeVarBytes(&buf, sct.Extensions, ExtensionsLengthBytes); err != nil {
+		return nil, err
+	}
+	sig, err := MarshalDigitallySigned(sct.Signature)
+	if err != nil {
+		return nil, err
+	}
+	if err := binary.Write(&buf, binary.BigEndian, sig); err != nil {
+		return nil, err
+	}
+	return buf.Bytes(), nil
+}
+
+// SerializeSCT serializes the passed in sct into the format specified
+// by RFC6962 section 3.2
+func SerializeSCT(sct SignedCertificateTimestamp) ([]byte, error) {
```

Comment:
It may well do (as you saw I did just that with the DS serialisation), I'm planning to go back through here once I've finished with this round of additions and normalise the naming a bit anyway, so I might extend that to normalising the API style a bit too.